assets/js/scripts-admin.js

```diff
@@ -575,23 +575,39 @@ jQuery( function( $ ) {
 	}
 
 	/**
-	 * Install and activate Optimole if the checkbox is checked.
+	 * Install and activate recommended plugins if the checkboxes is checked.
 	 */
-	function handleOptimole() {
+	function handlePlugins() {
 		const optimoleCheckbox = $( '#wizard-optimole-checkbox' );
+		const hyveCheckbox = $( '#wizard-hyve-checkbox' );
+
+		let promiseChain = Promise.resolve();
 
 		if ( optimoleCheckbox.length && optimoleCheckbox.is( ':checked' ) ) {
-			if ( ! wpmmVars.isOptimoleInstalled ) {
-				return installPlugin( 'optimole-wp' )
-					.then( () => {
+			promiseChain = promiseChain
+				.then(() => {
+					if ( ! wpmmVars.isOptimoleInstalled ) {
+						return installPlugin( 'optimole-wp' ).then( () => activatePlugin( 'optimole-wp' ) );
+					} else if ( ! wpmmVars.isOptimoleActive ) {
 						return activatePlugin( 'optimole-wp' );
-					} );
-			} else if ( ! wpmmVars.isOptimoleActive ) {
-				return activatePlugin( 'optimole-wp' );
-			}
+					}
```

Comment:
Just a small NIT. If you use `return`, you do not need to use the `else if`. Just: 

```js
if ( ! wpmmVars.isOptimoleInstalled ) {
    return installPlugin( 'optimole-wp' ).then( () => activatePlugin( 'optimole-wp' ) );
} 

if ( ! wpmmVars.isOptimoleActive ) {
    return activatePlugin( 'optimole-wp' );
}
```